app/src/main/res/values/third_party_dependencies.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I missed this before, but can we actually just make these example deps rather than real ones with real licenses? If the purpose of this is for testing, then we shouldn't even hint at real dependencies (especially since we don't want people checking in real license copies, hence the check in your other PR).

Reply:
done.